skopt/space.py

```diff
@@ -51,7 +61,7 @@ def fit(self, X):
         * `X` [array-like, shape=(n_categories,)]:
             List of categories.
         """
-        self._lb.fit(X)
+        self._lb.fit(encode_none(X))
```

Comment:
If you want, but I am fine with duplicating this very small piece of code (a single line!). I find it easier to read.